cyclist/src/edu/utexas/cycic/Cycic.java

```diff
@@ -273,8 +273,7 @@ public void handle(DragEvent event){
 					facility.cycicCircle = CycicCircles.addNode((String)facility.name, facility);
 					facility.cycicCircle.setCenterX(event.getX());
 					facility.cycicCircle.setCenterY(event.getY());
-					facility.cycicCircle.text.setLayoutX(event.getX()-facility.cycicCircle.getRadius()*0.7);
-					facility.cycicCircle.text.setLayoutY(event.getY()-facility.cycicCircle.getRadius()*0.6);
+                    VisFunctions.placeTextOnCircle(facility.cycicCircle,"bottom");
```

Comment:
it looks like there is some indentation discrepancies here. I agree that in general these files could use a linter, but for now we should probably stay with what's there.